src/modules/item/SectionHeader.tsx

```diff
@@ -21,25 +21,32 @@ const SectionHeader = ({ item }: SectionHeaderProps): JSX.Element => {
   });
   return (
     <Stack direction="column" spacing={2}>
-      <Stack direction="row" spacing={2} alignItems="center">
-        <Thumbnail
-          maxWidth="96px"
-          maxHeight="96px"
-          url={thumbnailSrc}
-          alt={item.name}
-          sx={{ borderRadius: 5 }}
-        />
+      <Stack
+        direction="row"
+        spacing={2}
+        alignItems="center"
+        justifyContent="space-between"
+      >
         <Stack>
-          <Typography className={FOLDER_NAME_TITLE_CLASS} variant="h2">
-            {item.name}
-          </Typography>
-          <Typography variant="caption">
-            {t(PLAYER.ITEM_TITLE_UPDATED_AT, {
-              date: formatDate(item.updatedAt, {
-                locale: i18n.language,
-              }),
-            })}
-          </Typography>
+          <Thumbnail
+            maxWidth="96px"
+            maxHeight="96px"
+            url={thumbnailSrc}
+            alt={item.name}
+            sx={{ borderRadius: 5 }}
+          />
+          <Stack>
+            <Typography className={FOLDER_NAME_TITLE_CLASS} variant="h2">
+              {item.name}
+            </Typography>
+            <Typography variant="caption">
+              {t(PLAYER.ITEM_TITLE_UPDATED_AT, {
+                date: formatDate(item.updatedAt, {
+                  locale: i18n.language,
+                }),
+              })}
+            </Typography>
+          </Stack>
```

Comment:
I think you did not mean to change this code. It broke the header:
Before:
<img width="652" alt="Screenshot 2024-05-24 at 11 10 49" src="https://github.com/graasp/graasp-player/assets/39373170/6a4631de-d073-49b4-9ff1-189ff4f48fbf">
Now: 
<img width="580" alt="Screenshot 2024-05-24 at 11 10 09" src="https://github.com/graasp/graasp-player/assets/39373170/15dd4690-9112-4944-b32e-d2e2efa0df76">